datasets/gov/hi/vaistu_suvartojimas/antimikrobinis.csv

```diff
@@ -0,0 +1,27 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/hi/vaistu_suvartojimas,,,,,,,,,,,,Antimikrobinių vaistinių preparatų suvartojimo Lietuvoje duomenys,
+,,,,,,,,,,,,,,
+,,,,Antimikrobinis,,,,,,3,,,Antimikrobinių vaistinių preparatų suvartojimas,
+,,,,,metai,integer,yr,,,2,open,,Antimikrobinių vaistinių preparatų suvartojimo statistikos vedimo metai,
+,,,,,istaiga,ref,/datasets/gov/vaspvt/licencijos/Istaiga[ja_kodas],,,4,open,,Sveikatos priežiūros įstaigos JAR kodas,
+,,,,,ligonine,string,,,,2,open,,Ligoninės pavadinimas,
+,,,,,tipas,string,,,,4,open,,Ligoninės tipas,"Ligoninių tipai: bendrojo pobūdžio ligoninė, palaikomojo gydymo ir slaugos ligoninė, specializuota ligoninė"
+,,,,,atc,string,,,,4,open,,Tarptautinę anatominio terapinio cheminio indekso klasifikacijos kodas,
+,,,,,veiklioji,string,,,,4,open,,Vaistinės veikliosios medžiagos pavadinimas,
+,,,,,komercinis,string,,,,4,open,,Vaistinio preparato komercinis pavadinimas,
+,,,,,pakid,string,,,,4,open,,Antimikrobinio vaistinio preparato pakuotės identifikavimo kodas,"Reikšmę turi tik tiek preparatai, kurie yra registruoti Europos vaistų agentūros"
+,,,,,stiprumas,string,,,,4,open,,Antimikrobinio vaistinio preparato pakuotės stiprumo reikšmė(-s),
+,,,,,vnt,string,,,,4,open,,Antimikrobinio vaistinio preparato stiprumo vienetai,
+,,,,,,enum,,,'g',,,,Gramai,
+,,,,,,,,,'iu',,,,Tarptautiniai vienetai,
+,,,,,,,,,'mg',,,,Miligramai,
+,,,,,,,,,'mu',,,,Tarptautinio vieneto milijoninės dalys,
+,,,,,,,,,'ud',,,,Antimikrobinio vaistinio preparato techninio vieneto dozė,
+,,,,,gramas,number,g,,,4,open,,Antimikrobinio vaistinio preparato pakuotės stiprumo reikšmė(-s) gramais,
+,,,,,pso_vtd,number,g,,,4,open,,Vaisto vidutinė terapinė paros dozė,
+,,,,,suvartota,integer,,,,4,open,,Suvartotų pakuočių skaičius,
+,,,,,pcs,string,,,,4,open,,Antimikrobinio vaistinio preparato technininis vienetas ir/arba vienetų skaičius ,
+,,,,,tech_vnt,integer,,,,4,open,,Suvartota techninių vienetų iš pakelio,
+,,,,,forma,string,,,,4,open,,Antimikrobinio vaistinio preparato farmacinė forma,
+,,,,,vtd_1000_pacientu,number,U,,,4,open,,"Nustatytas vaisto vartojimo techninis vienetas, išreiškiamas gramais (ar miligramais) per parą ir skaičiuojamas 1000-čiui pacientų.","VTD – tai nustatytas vaisto vartojimo techninis vienetas, išreiškiamas gramais (ar miligramais) per parą ir kaip rodiklis skaičiuojamas 1000-čiui pacientų."
+,,,,,vtd_1000_lovadieniu,number,U,,,4,open,,"Nustatytas vaisto vartojimo techninis vienetas, išreiškiamas gramais (ar miligramais) per parą ir skaičiuojamas 1000-čiui lovadienių.","VTD – tai nustatytas vaisto vartojimo techninis vienetas, išreiškiamas gramais (ar miligramais) per parą ir kaip rodiklis skaičiuojamas 1000-čiui lovadienių."

```

Comment:
Išreiškiamas gramais, tai nėra kažkoks vienetų skaičius, tai abejoju, kad čia reikia U

Reply:
Diskusijų klausimas, bet parašysiu g. Jeigu vartotojams bus neaišku, pataisysime, nes dozė gali būti išreikšta ir gramais, ir miligramais, be to, tai santykinis, normalizuotas vienetas.